doc/SpectatorView.Native.md

```diff
@@ -1 +1,77 @@
+- **SpectatorView.OpenCV.dll** is used in both ArUco marker detection and DSLR camera calibration.
+
+# SpectatorView.Compositor.dll & SpectatorView.Compositor.UnityPlugin.dll
+
+### 1. Obtain external dependencies
+**DeckLink Capture Card**
+If you are using a Blackmagic capture card, you will need to install the SDK and create a Visual Studio user macro for its location.
++ Download the DeckLink SDK from here: https://www.blackmagicdesign.com/support - Search for Desktop Video SDK in "Latest Downloads"
++ Extract the SDK anywhere on your computer.
++ Update the DeckLink_inc user macro in [dependencies.props](../src/SpectatorView.Native/SpectatorView.Compositor/dependencies.props) with the corresponding path on your computer.
++ Restart Visual Studio
+
+**Elgato Capture Card**
+If you are using an Elgato capture card, you will need to clone Elgato's [gamecapture github repo](https://github.com/elgatosf/gamecapture).
+- Open a Command Prompt in administrator mode
+- Navigate to a folder in which you would like to store your repositories (ex: c:\git)
+- git clone <https://github.com/elgatosf/gamecapture>
+- Update the Elgato_Filter user macro in [dependencies.props](../src/SpectatorView.Native/SpectatorView.Compositor/dependencies.props) with the corresponding path on your computer.
+- Restart Visual Studio
+
+### 2. Build the Plugins
+- Open [SpectatorView.Native.sln](../src/SpectatorView.Native/SpectatorView.Native.sln) in visual studio.
+- Build a **Release x64** version of SpectatorView.Compositor.dll with [**SpectatorView.Compositor**](../src/SpectatorView.Native/SpectatorView.Compositor/Compositor/SpectatorView.Compositor.vcxproj)
+- Build a **Release x64** version of SpectatorView.Compositor.UnityPlugin.dll with [**SpectatorView.Compositor.UnityPlugin**](../src/SpectatorView.Native/SpectatorView.Compositor/UnityPlugin/SpectatorView.Compositor.UnityPlugin.vcxproj)
+
+# SpectatorView.OpenCV.dll
+>Note: SpectatorView.OpenCV.dll introduces dependencies on OpenCV. OpenCV does not have a MIT license. For more information on OpenCV's license, see [here](https://opencv.org/license/). 
+* **DSLR camera calibration** requires a **Release x64** version of this binary built from the [**SpectatorView.OpenCV.Desktop**](../src/SpectatorView.Native/SpectatorView.OpenCV/Desktop/SpectatorView.OpenCV.Desktop.vcxproj) visual studio project.
+* **ArUco Marker detection** on a HoloLens 1 device requires a **Release x86** version of this binary built from the [**SpectatorView.OpenCV.UWP**](../src/SpectatorView.Native/SpectatorView.OpenCV/UWP/SpectatorView.OpenCV.UWP.vcxproj)
+
+### 1. Install [Vcpkg](https://github.com/microsoft/vcpkg)
+
+- Open a Command Prompt in administrator mode
+- Navigate to a folder in which you would like to store your repositories (ex: c:\git)
+- git clone <https://github.com/Microsoft/vcpkg>
+- cd vcpkg
+- git checkout 05b31030cee412118a9710daf5b4652a684b7f50

```

Comment:
>05b31030cee412118a9710daf5b4652a684b7f50 [](start = 15, length = 40)

Did you re-test this commit SHA? When I installed OpenCV I had to use the latest version of the master branch, but I'm not sure what commit SHA that was.

Reply:
I think I will change this to just say to checkout master